src/subscript/merge_rft_ertobs/merge_rft_ertobs.py

```diff
@@ -59,13 +67,58 @@ def get_parser():
     return parser
 
 
+def split_wellname_reportstep(wellname_reportstep):
+    """Split a string that might contain both a wellname and a report step,
+    at least it should contain a wellname.
+
+    The reportstep is a number at the end, following an underscore.
+
+    This function is needed and complex due differing standards on whether
+    the report step should be present in the filename or not.
+
+    Reporsteps larger than 9 is not supported, this is a compromise.
+
+    Examples::
+
+        "F_A-3" gives (F_A-3, 1)
+        "F_A-4_1" gives (F_A-4, 1)
+        "F_A-4_2" gives (F_A-4, 2)
+        A-4 gives (A-4, 1)
+        A-5_99 gives (A-5_99, 1)  # report steps more than 10 not supported.
+        "R_A4_1" gives (R_A4, 1)
+        "R_A4" gives (R_A4, 1)
+        "R_A_4" gives (R_A, 4)  # Warning, this is probaly unintended!
+
+    Args
+        wellname_reportstep (str)
+
+    Returns
+        tuple: wellname and reportstep. Reportstep defaulted to 1 if not found
```

Comment:
Why 1 and not None?

Reply:
It must/should match the default number in use in the ert config, which is never None.